setup.cfg

```diff
@@ -39,7 +39,7 @@ install_requires =
     six
     pypiwin32;platform_system=="Windows"
     PyYAML;python_version>="2.7"
-    PyYAML<4;python_version<"2.7"    
+	PyYAML<4;python_version<"2.7"
```

Comment:
Missing a space here...

Reply:
silly vscode 😞 - using tabs instead of spaces